src/components/atoms/MultiSelect/Select/Select.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { ArrowIcon } from 'assets/index';
```

Comment:
```suggestion
import { ArrowIcon } from 'assets';
```

/index is not needed, TS looking for index file by its own